homeassistant/components/tuya/select.py

```diff
@@ -307,6 +307,13 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Early Product "QT" device (for Pool Chlorinator)
+    "qt": (
```

Comment:
As far as I am aware, we shouldn't be adding "qt" devices. They can be anything, are considered legacy, or unsupported by Tuya.